.evergreen/config_generator/components/sanitizers/asan_cse.py

```diff
@@ -11,13 +11,15 @@
 # fmt: off
 COMPILE_MATRIX = [
     ('ubuntu1804', 'clang', None, ['cyrus']),
+    ('ubuntu2004', 'clang', None, ['cyrus']),
 ]
 
 TEST_MATRIX = [
     ('ubuntu1804', 'clang', None, 'cyrus', ['auth'], ['server',          ], ['4.2', '4.4', '5.0', '6.0'                ]),
```

Comment:
Since these entries are not grouped together, suggest simplifying the formatting.
```python
TEST_MATRIX = [
    ('ubuntu1804', 'clang', None, 'cyrus', ['auth'], ['server'], ['4.2', '4.4', '5.0', '6.0']),

    # Test 7.0+ with a replica set since Queryable Encryption does not support the 'server' topology. Queryable Encryption tests require 7.0+.
    # Test 7.0+ with Ubuntu 20.04+ since MongoDB 7.0 no longer ships binaries for Ubuntu 18.04.
    ('ubuntu2004', 'clang', None, 'cyrus', ['auth'], ['server', 'replica'], [                           '7.0', 'latest']),
]
```